packages/react-instantsearch/src/components/StarRating.js

```diff
@@ -49,6 +49,32 @@ class StarRating extends Component {
         )}
         />);
     }
+    if (isLowest && selected) {
```

Comment:
Did you try to merge both this code path with the normal one? I am wondering if this could be a bit less verbose and only changes a into div + avoid setting onClick and href props.

Something like

```js
buildItem('a', commonProps, {href: ...});
buildItem('div', commonProps);
```